LICENSE

```diff
@@ -1,182 +1,181 @@
-
                                  Apache License
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
Please revert the indention changes to this license. It makes it difficult to see what was actually changed.